src/components/vote-form.js

```diff
@@ -1,8 +1,55 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import CandidateForm from "./candidate-form";
+
 import styled from "styled-components";
+import axios from "axios";
+
+export default function VoteForm(props) {
+  const [candidateList, setCandidateList] = useState([]);
+  const { name, voteCount, rank, id } = candidateList;
+
+  useEffect(() => {
+    getCandidateList();
+  }, [candidateList]);
 
-export default function VoteForm() {
-  return <Wrapper>안녕 나는 투표 폼!</Wrapper>;
+  const getCandidateList = async () => {
+    const data = await axios
+      .get(process.env.API_HOST + "/candidates/")
+      .then(function (response) {
+        return response.data;
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+    setCandidateList(data);
+  };
+
+  var i = 1;
+
+  return (
+    <Wrapper>
+      <Header1>
+        <Strong>프론트앤드 인기쟁이는</Strong> 누구?
+      </Header1>
+      <Header2>CEOS 프론트엔드 개발자 인기 순위 및 투표 창입니다.</Header2>
+      <CandidateListWrapper>
+        {candidateList
+          .sort((a, b) => {
+            return b.voteCount - a.voteCount;
+          })
```

Comment:
```suggestion
          .sort((a, b) => b.voteCount - a.voteCount)
```
화살표함수에서 바로 return할땐 중괄호와 return문을 생략할 수 있습니다~